site/app/templates/grading/electronic/Details.twig

```diff
@@ -712,8 +712,7 @@
     }
 
     function changeSortOrder() {
-        const status = {{ sort == 'random' ? 1 : 0}};
-        if (status) {
+        if ("{{ sort == 'random' ? true : false }}") {
```

Comment:
Is there a reason why you put quotes around it here?  You should just be able to put the JS constants `true` and `false` there instead of relying upon string inference.

Reply:
ah, I think the issue is that the Twig constants `true` and `false` are used.  Try this:
`if ({{ sort == 'random' ? "true" : "false" }}) {`